addons/stock/migrations/10.0.1.1/post-migration.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# © 2017 Trescloud <http://trescloud.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    update_picking_type_id(env)
+    update_ordered_qty(env)
+    populate_stock_scrap(env)
+    
+def update_picking_type_id(env):
+    '''
```

Comment:
According PEP257, method documentation goes with triple double quota, not simple, and first line start in the same one as the initial quotas.